tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -458,38 +451,36 @@ describe('Target aggregates', () => {
         { id: 'b_target', title: 'the most target', progressBar: true, counter: '27%' },
       ];
 
-      await helper.takeScreenshot('detail-targets.png');
       await expectTargets(expectedTargets);
       await openTargetDetails(expectedTargets[0].id);
       await clickOnTargetAggregateListItem(clarissa._id);
 
-      expect(await element(by.css('.content-pane .meta h2')).getText()).toEqual('Clarissa');
+      expect(await $('.content-pane .meta h2').getText()).to.equal('Clarissa');
```

Comment:
Same as previous comments about selectors definitions. Can you put all selectors definitions in this documents into the corresponding page-object file, please?